src/parseOption.ts

```diff
@@ -132,7 +142,7 @@ export function prepareAllFieldsData(itemData: ScenegraphData, options: Option =
   const fieldOptions: TemporaryFieldOption = {};
   if (options && options.fields) {
     for (const optionField of options.fields) {
-      fieldOptions[optionField.field] = optionField;
+      fieldOptions[optionField.field as string] = optionField;
```

Comment:
I wonder if this casting here may cause a bug as field can be a function here?

Reply:
Ah, you're right, sorry -- I misunderstood how `options.fields` is used when `showAllFields === true`. Will add a type check and warning.

Reply:
Reading closer, correction -- will call `field()` to generate a string to use as the key.